templates/cohorts/cohort_details.html

```diff
@@ -241,20 +241,20 @@ <h4 class="panel-title">Selected Filters</h4>
                 <h4 class="panel-title">Details</h4>
             </div>
             <div class="panel-body">
-                <div class="single-line">
+                <div class="row col-md-12 row-spacer-v">
```

Comment:
It does, but this is the cohort_details page, which has always had each detail on its own line. Do we want to change that?

The new_cohort page's details section is single-line, with sample and participant counts on the same line, because a new cohort doesn't have permissions or revisions to list, and the ownership is implicit. 

Swapped to one of those spacer classes.